zbx_redis_stats.py

```diff
@@ -120,13 +120,14 @@ def default():
 
 	a = []
 	for i in server_info:
-		a.append(Metric(args.hostname, ('redis[%s]' % i), server_info[i]))
+		a.append(Metric(hostname, ('redis[%s]' % i), server_info[i]))
 
 	keys = client.keys('*')
 	llensum = 0
 	for key in keys:
-		llensum += client.llen(key)
-	a.append(Metric(args.hostname, 'redis[llenall]', llensum))
+                if client.type(key) == 'list':
+                        llensum += client.llen(key)
+	a.append(Metric(hostname, 'redis[llenall]', llensum))
```

Comment:
The same `hostname` issue as previous.
And we should check the type of `key` before calling `llen(key)`.